src/toxcore/stats.rs

```diff
@@ -23,73 +20,33 @@ impl Stats {
     }
 }
 
-#[cfg(target_pointer_width = "64")]
 #[derive(Default)]
-/// Struct for counting packets on x64 CPU.
```

Comment:
Take a look at warnings: https://travis-ci.com/tox-rs/tox/jobs/241932457#L382

At least this should be smt like: `/// Struct for counting packets` to please the compiler